Documentation/usingupm.md

```diff
@@ -0,0 +1,74 @@
+# Mixed Reality Toolkit and Unity Package Manager
+
+Starting with version 2.5.0, the Microsoft Mixed Reality Toolkit is available using the Unity Package Manager (UPM), on Unity 2019.4 and newer. Using UPM, the MRTK gains dependency management and a simple update path, as will be described in the following sections.
```

Comment:
```suggestion
Starting with version 2.5.0, the Microsoft Mixed Reality Toolkit is available using the Unity Package Manager (UPM), on Unity 2019.4 and newer.
```

Reply:
Since this is going to be a longer-running doc, this part doesn't really add a ton of value for folks after the fact.